src/trainingarena/TrainingArena.js

```diff
@@ -53,21 +78,16 @@ class TrainingArena extends React.Component<{||}> {
 				/>
 				<div className="column taleft">
 					<h5>Choose your Tank, Commander</h5>
-					<select className="dropdownMenu">
-						<option defaultValue>Select a Tank</option>
-						<option value="1">Child Consumer</option>
-						<option value="2">Fast Bang</option>
-						<option value="3">Biggest Gun</option>
-					</select>
-					</div>
+					{
+						this.state.selectedTank==null?<div></div>:
+						<TankDisplay tankToDisplay={this.state.selectedTank} />
+					}
+					<SelectTank
+						allTanks={this.state.allTanks}
+						changeSelectedTank={(tank) => {}}

```

Comment:
Are these gonna be updated later? Currently I can't change the selected tank in Battle Arena and Training Arena.